src/bioregistry/data/bioregistry.json

```diff
@@ -32814,8 +32820,14 @@
       "name": "Charles Tapley Hoyt",
       "orcid": "0000-0003-4423-4370"
     },
-    "description": "ICD-10-CM is the official system of assigning codes to diagnoses and procedures associated with hospital utilization in the United States.",
+    "description": "ICD-10-CM is the official system of assigning codes to diagnoses and procedures associated with hospital utilization in the United States issued by the National Center for Health Statistics (NCHS) and Centers for Medicare & Medicaid Services (CMS).",
     "example": "F4521",
+    "example_extras": [
+      "C34.90",
+      "C4A.8",
+      "C91.Z",
+      "C91.Z0"
+    ],
```

Comment:
you can also add a field `"example_decoys"` that has negative examples

Reply:
super cool idea